icicle/include/icicle/utils/modifiers.h

```diff
@@ -16,15 +16,18 @@
   #define DEVICE_INLINE      __device__ INLINE_MACRO
   #define HOST_DEVICE        __host__ __device__
   #define HOST_DEVICE_INLINE HOST_DEVICE INLINE_MACRO
+  #define ALIGN(size)        __align__(size)
 #else // not CUDA
   #define INLINE_MACRO
   #define UNROLL
-  #define HOST_INLINE
+  #define HOST_INLINE inline
   #define DEVICE_INLINE
```

Comment:
why not defined as inline

Reply:
@yuvalingo ? you made this change, should it be reverted?